engine/classes/ElggEntity.php

```diff
@@ -2148,8 +2148,10 @@ public function setLocation($location) {
 	 * @todo Unimplemented
 	 */
 	public function setLatLong($lat, $long) {
-		$this->set('geo:lat', $lat);
-		$this->set('geo:long', $long);
+		$lat_md = "geo:lat";
+		$long_md = "geo:long";
+		$this->$lat_md = $lat;
```

Comment:
Wasn't sure which PHP version this introduced in.
On Feb 26, 2015 4:29 PM, "Paweł Sroka" <notifications@github.com> wrote:

> In engine/classes/ElggEntity.php
> <https://github.com/Elgg/Elgg/pull/8007#discussion_r25434904>:
>
> > @@ -2148,8 +2148,10 @@ public function setLocation($location) {
> >  	 * @todo Unimplemented
> >  	 */
> >  	public function setLatLong($lat, $long) {
> > -		$this->set('geo:lat', $lat);
> > -		$this->set('geo:long', $long);
> > +		$lat_md = "geo:lat";
> > +		$long_md = "geo:long";
> > +		$this->$lat_md = $lat;
>
> Why not $this->{"geo:lat"} = $lat ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8007/files#r25434904>.
>